symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/RealTimeEventType.java

```diff
@@ -0,0 +1,107 @@
+package com.symphony.bdk.core.service.datafeed.impl;
+
+import com.symphony.bdk.core.service.datafeed.RealTimeEventListener;
+import com.symphony.bdk.gen.api.model.V4Event;
+import com.symphony.bdk.gen.api.model.V4Payload;
+import io.vavr.Function3;
+
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+/**
+ * Enumeration of possible types of Real Time Events that can be retrieved from the DataFeed.
+ * More information : https://rest-api.symphony.com/reference#real-time-events-v4
+ */
+enum RealTimeEventType {
+
+    MESSAGESENT((listener, event, innerEvent) -> {
+        listener.onMessageSent(event);
```

Comment:
As I mentioned earlier, I don't think that we should consider providing the whole `V4Event` to listener callbacks, this might introduce more confusion to end-developers as well as making our API more difficult to understand. 

The goal of the BDK is really to hide the complexity of our REST API, and this specifically one point that we need to take care of. 

WDYT?

Reply:
Also (and assuming that we would keep this strategy), the `innerEvent` is already part of the `event` so there is no really need to have it part of the lambda function.

Reply:
So I think it's better if we keep just one callbacks take the initiator and the dedicated event as inputs. I will remove the other.